include/uapi/sound/sof/abi.h

```diff
@@ -28,7 +28,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 21
+#define SOF_ABI_MINOR 22
```

Comment:
@dbaluta looks like this can be squashed with the previous patch right?

Reply:
@ranj063 I've seen previous changes preferred this approach of having a separate patch. I wouldn't mind either.

@plbossart what do you think?

Reply:
"Older FW will work with newer kernels as they will directly ignore ext_data_length field."

that's true, but functionally the parameters will be ignored and the result is undefined. If the firmware version is < 3.22 you probably don't want to even such an IPC at all, and probably want to prevent this path from being created in the first place.